### DISCUSSION TITLE
I had to reinstall HA having issues with Mail and Packages

### DISCUSSION THREAD

**Discussion by crankycowboy** [moralmunky/Home-Assistant-Mail-And-Packages#825] [General]
I'm trying to reinstall mail and packages and in HACs, I can't see any beta versions anymore. Clicking the radio button to switch to beta does nothing. Any suggestion please?

**Comment by firstof9** [comment]
There hasn't been a beta release in some time, use the latest regular release.

**Reply by crankycowboy** [reply]
Well I have installed that version, and everything configures successfully with the integration, but I just keep getting an error where it says "initializing" and then says "error" and keeps repeating that. I am certain that the imap info is correct. Any other suggestion?
![m P error](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/2176738/1ffe3cf8-dd2b-43b8-a53b-46dafad413cf)

**Reply by firstof9** [reply]
Check your `home-assistant.log` file for the error message.

**Reply by crankycowboy** [reply]
Here is the message from the logs:

This error originated from a custom integration.

Logger: custom_components.mail_and_packages
Source: custom_components/mail_and_packages/__init__.py:103
Integration: Mail and Packages (documentation, issues)
First occurred: May 15, 2023 at 6:43:13 PM (601 occurrences)
Last logged: 1:01:56 PM

Error updating sensor data:

**Reply by crankycowboy** [reply]
So I uninstalled the HACS integration and also uninstalled the HA integration. I noticed the "mail and packages" folder was still there under "custom components" so I deleted the folder and started from scratch again. It's still behaving the same. This time, I'm getting some different errors. The entities never even get installed.  Any other suggestions of what I may try?
![mail error2](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/2176738/66a2ab5d-9150-4db2-bf98-973b1a79a4c8)
![mail error1](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/2176738/39103f3a-d6c9-41d7-b119-9bf40b3a22ff)
![mail error 3](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/2176738/c887eb50-9345-4f78-9805-0201b5fb81e6)

**Reply by firstof9** [reply]
These are screenshots and not the errors from the `home-assistant.log` file.

**Reply by crankycowboy** [reply]
I just went into settings>system>logs to get that info. Where else do I need to go to get what you are asking for?

**Reply by firstof9** [reply]
The `home-assistant.log` file exists in the `/config` directory with your yaml files.

**Comment by appsfinder2** [comment]
> I'm trying to reinstall mail and packages and in HACs, I can't see any beta versions anymore. Clicking the radio button to switch to beta does nothing. Any suggestion please?

did you click redownload the there a toggle to show beta 

![HACS_–_Home_Assistant](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/31155587/6e2323a5-14f5-4e0f-b793-5274fe30ffef)
![HACS_–_Home_Assistant-2](https://github.com/moralmunky/Home-Assistant-Mail-And-Packages/assets/31155587/381897d9-5f00-425d-8c98-165759b28b8a)
